src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,49 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    public static final int STORAGE_SIZE = 10;
+    private int size = 0;
+    private Object[] keys;
+    private Object[] values;
```

Comment:
Use parameterized type
```suggestion
    private K [] keys;
    private V [] values;
```